Fantasy-Quest/Assets/Scripts/Scene/NewScene/EnterNewScene.cs

```diff
@@ -0,0 +1,96 @@
+using System.Collections;
+using System.Collections.Generic;
+using Interaction.Item;
+using Sirenix.OdinInspector;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Scene.NewScene
+{
+    [AddComponentMenu("Scripts/Scene/NewScene/Scene.NewScene.EnterNewScene")]
+    internal class EnterNewScene : MonoBehaviour, ISceneTransition
+    {
+        [Required]
+        [SerializeField]
+        private GameObject cat;
+
+        [SerializeField]
+        private List<ParticleSystem> effects = new();
+
+        [SerializeField]
+        private string newScene;
+
+        [SerializeField]
+        private bool isFiringInParallel = false;
+
+        //[SerializeField]
+        //private bool afterAllEffects = false;
+
+        [ShowIf("isFiringInParallel")]
+        [SerializeField]
+        private bool afterFirstEffect = false;
+
+        private float duration = 0f;
+
+        public void ToNewScene()
+        {
+            if (isFiringInParallel)
+            {
+                _ = StartCoroutine("GoInParallel");
+            }
+            else
+            {
+                _ = StartCoroutine("GoOneByOne");
```

Comment:
```suggestion
                _ = StartCoroutine(GoOneByOne());
```
Use this version is better than using name of method. Because when you rename method, the string will not be changed